packages/lib/assets/css/marketing.css

```diff
@@ -103,7 +103,7 @@
   }
 }
 
-@media screen and (min-width >= 768px) {
+@media screen and (width >= 768px) {
```

Comment:
Was this intended?

Reply:
stylelint threw an error on it

```
> @balancer/frontend-monorepo@ stylelint /home/groninge/projects/frontend-monorepo
> stylelint '**/*.css'


packages/lib/assets/css/marketing.css
  106:18  ✖  Unexpected invalid media query " (min-width >= 768px)", "min-" and "max-" prefixes are not needed when using range queries  media-query-no-invalid
```

Reply:
@uiuxxx is this linting change valid? or will it cause problems.